src/sync/Synchroniser.js

```diff
@@ -84,24 +94,41 @@ export class Synchroniser {
 
   /**
    * Return whether the synchroniser has been completely or partially initialised.
-   * @return {string} Either 'complete', 'partial', 'uninitialised'
+   * @return {string} Either 'complete', 'partial', 'uninitialised' TODO: make sure this is corrected
```

Comment:
Part of me was hoping this would slip through haha. I'm pretty sure it's safe to change to `either 'true' or 'false'`, though I considered using this tweaked instead of  creating `SYNC_IS_SYNCED`.